Hangman/Board.py

```diff
@@ -0,0 +1,43 @@
+class Board:
+    def __init__(self, word="default"):
+        self.chance = 6  # Starting Chances
+        self.guessed_word = "-" * len(word)
+        self.guessed_letters = []
```

Comment:
we could use a better data structure here than list

Reply:
i will use set instead